gui/slick/js/new/comingEpisodes.js

```diff
@@ -38,8 +38,3 @@ $(document).ready(function(){
         });
     }
 });
-
-setTimeout(function () {
-    "use strict";
-    location.reload(true);
```

Comment:
If you need auto refresh install an extension for your browser or hit F5, this causes more problems than it solves and honestly I doubt 99% of users want it. I for one get annoyed as fuck when it randomly refreshes as I'm looking at the page.